bazel/ray_deps_setup.bzl

```diff
@@ -238,8 +238,8 @@ def ray_deps_setup():
     auto_http_archive(
         name = "com_github_grpc_grpc",
         # NOTE: If you update this, also update @boringssl's hash.
-        url = "https://github.com/grpc/grpc/archive/refs/tags/v1.45.2.tar.gz",
-        sha256 = "e18b16f7976aab9a36c14c38180f042bb0fd196b75c9fd6a20a2b5f934876ad6",
+        url = "https://github.com/grpc/grpc/archive/refs/tags/v1.48.0.tar.gz",
```

Comment:
1.48.0 is yanked in pypy. does this matter? can use https://github.com/grpc/grpc/releases/tag/v1.48.2